elasticai/creator/nn/quantized_grads/quantize.py

```diff
@@ -0,0 +1,9 @@
+from jedi.inference.gradual.typing import Protocol
+
+from elasticai.creator.nn.quantized_grads.quantization_config import QuantizationConfig
+
+
+class Quantize(Protocol):
+    config: QuantizationConfig
```

Comment:
Having that field here seems odd. Can't we just call the quantize function from outside so that it quantizes? What i mean is, is there a use case, where another caller has to access that config? If so, then why are we not defining anything else about it? I mean there's nothing a caller could do with `QuantizationConfig` except for passing it around, because the corresponding dataclass is empty.